cpanfile

```diff
@@ -1,11 +1,12 @@
 requires 'perl', '5.014';
 
-requires 'Data::Chronicle::Reader';
-requires 'Data::Chronicle::Writer';
+requires 'Data::Chronicle::Reader', '>= v0.18';
+requires 'Data::Chronicle::Writer', '>= v0.18';
```

Comment:
We can probably just write `requires 'Data::Chronicle', '>= 0.18';` here as I don't think we'll be spinning off the Reader and Writer into separate CPAN dists anytime soon.

(The `v0.18` is also incorrect, as that would be expanded by [`version`](https://metacpan.org/pod/version) as `0.018_000`.)

```
% reply -Mversion
0> version->parse("0.18") == version->parse("v0.18") or "they're not equal"
$res[0] = "they're not equal"
```